python/pyspark/sql/context.py

```diff
@@ -29,9 +29,10 @@
 from pyspark.sql.readwriter import DataFrameReader
 from pyspark.sql.streaming import DataStreamReader
 from pyspark.sql.types import IntegerType, Row, StringType
+from pyspark.sql.udf import UDFRegistration
```

Comment:
I intendedly kept this to retain the import path `pyspark.sql.context.UDFRegistration` just in case.